static/app/components/button.tsx

```diff
@@ -17,6 +17,8 @@ import {Theme} from 'sentry/utils/theme';
  */
 type ButtonElement = HTMLButtonElement & HTMLAnchorElement & any;
 
+type TooltipProps = React.ComponentProps<typeof Tooltip>;
```

Comment:
@evanpurkhiser in this case it's slightly different though. Iirc because you are creating an alias, you are  adding some overhead, because the type still has to be stored somewhere compared to having a static reference to the TooltipProps